src/classBased/components/TodosList.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
[OPTIONAL]
- Avoid deactivating too many linters, this may lead to a weak code structure. Don't worry about it now, but keep it in mind for the next projects.